src/locales/en/translation.json

```diff
@@ -59,7 +62,7 @@
   "anyAddress": "Any Address",
   "anyToken": "any token",

```

Comment:
Since you removed this key from PermissionInfoLine I think you can remove from here too (only being used here)

Reply:
Removed, thanks